compiler/util/stringutil.cpp

```diff
@@ -56,6 +56,10 @@ const char* astr(const std::string& s)
 {
   return astr(s.c_str());
 }
+const char* astr(std::string_view s)
+{
+  return astr(s.data());
+}
```

Comment:
I don't think string_view guarantees null termination. If that's the case, the safer bet here is `return astr(std::string(s))`

Reply:
Oh good to know, I didn't realize `string_view` isn't null terminated. I'll make this change.
Will also add comments for the driver save/restore usages that it expects a null-terminated string, since I made that assumption there but I believe it's a safe enough assumption that we don't need to make a string copy.